colossalai/kernel/extensions/extension_builder.py

```diff
@@ -0,0 +1,243 @@
+# This code has been adapted from the DeepSpeed library.
+# Copyright (c) Microsoft Corporation.
+
+# Licensed under the MIT License.
+import importlib
+import os
+import time
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import List, Optional, Union
+
+from .utils import check_cuda_availability, check_system_pytorch_cuda_match, print_rank_0
+
+
+class ExtensionBuilder(ABC):
```

Comment:
Is this previous op builder?

Reply:
yes, just move it here

Reply:
Did you remove the previous code? I didn't find the deletion in this PR

Reply:
no, it is still needed by other op in op builders

Reply:
only delete multihead attention in cuda/mha